### ISSUE TITLE
Wrong results due to scoring

### ISSUE THREAD

**Issue by CodeRunRepeat** [microsoft/presidio#191]
This bug talks about an issue where a string is considered by two different recognizers to be a PII entity.
Its not bad as compleetly missing it however there are scenario (like the one described below) that the wrong entity type 'wins' due to incorrect scoring.

consider the following request to analyze:
```json
{
  "text": "my social number is 078051120 and my name is Jon doe",
  "analyzeTemplate": {
    "fields": [
      {
        "name": "PHONE_NUMBER"
      },
      {
        "name": "US_SSN"
      },
      {
        "name": "CREDIT_CARD"
      },
      {
        "name": "DATE_TIME"
      }
    ]
  }
}
```

The result we get for 078051120 is DATE_TIME with score 0.85
if we remove DATE_TIME from the template we get the correct result of US_SSN

Basically its ok because it will be matched anyway, altough the anonymize methods might be different.
in this specific case this ssn is given a WEAK score of 0.3 and due to the context it is improved to 0.6, which is still less than 0.85.
if the SSN would have been in the correct SSN format XXX-XX-XXXX it would have got a better score with context of 0.85 which is the same as date time and might still be the one removed.

This bug should be discussed. 
a) do we want to give regex results priority over spacy?
b) maybe we want to give context more affect on the score
c) maybe we want to allow the user to configure the right behaviour.

**Comment by navalev** [comment]
The api returns the score for each field. As DATE_TIME is a spacy field, it is not possible to change scoring. 
A possible solution is for the user to filter out fields either in the request, or from the result.
